testutil/integration/ibc/coordinator/coordinator.go

```diff
@@ -0,0 +1,121 @@
+// Copyright Tharsis Labs Ltd.(Evmos)
+// SPDX-License-Identifier:ENCL-1.0(https://github.com/evmos/evmos/blob/main/LICENSE)
+package coordinator
+
+import (
+	"testing"
+	"time"
+
+	ibctesting "github.com/cosmos/ibc-go/v7/testing"
+	"github.com/evmos/evmos/v15/testutil/integration/common/network"
+	ibcchain "github.com/evmos/evmos/v15/testutil/integration/ibc/chain"
+)
+
+// Coordinator is the interface that defines the methods that are used to
+// coordinate the execution of the IBC relayer.
+type Coordinator interface {
+	// IncrementTime iterates through all the TestChain's and increments their current header time
+	// by 5 seconds.
+	IncrementTime()
+	// UpdateTime updates all clocks for the TestChains to the current global time.
+	UpdateTime()
+	// UpdateTimeForChain updates the clock for a specific chain.
+	UpdateTimeForChain(chainID string)
+	// GetChain returns the TestChain for a given chainID.
+	GetChain(chainID string) ibcchain.Chain
+	// Setup constructs a TM client, connection, and channel on both chains provided. It will
+	// fail if any error occurs. The clientID's, TestConnections, and TestChannels are returned
+	// for both chains. The channels created are connected to the ibc-transfer application.
+	Setup(path *ibctesting.Path)
+	// CommitNBlocks commits n blocks on the chain with the given chainID.
+	CommitNBlocks(chainID string, n uint64) error
+}
+
+// TODO: Replace for a config
```

Comment:
do you want to address the TODO here or on a separate PR?

Reply:
Yes! As I start doing the outputs test to have a more clear idea of what's needed